charts/openshift-metering/templates/presto/hive-metastore-statefulset.yaml

```diff
@@ -120,6 +120,36 @@ spec:
 {{- end }}
         resources:
 {{ toYaml .Values.presto.spec.hive.metastore.resources | indent 10 }}
+{{- if .Values.presto.spec.hive.metastore.config.tls.enabled }}
+      - name: ghostunnel-server
+        env:
+        - name: HIVE_METASTORE_CA_CRTFILE
+          value: opt/hive/tls/ca.crt
+        - name: HIVE_METASTORE_KEYFILE
+          value: opt/hive/tls/tls.key
+        - name: HIVE_METASTORE_CRTFILE
+          value: opt/hive/tls/tls.crt
+        command: ["ghostunnel"]
+        args:
+        - server
+        - --listen
+        - localhost:8443
+        - --target
+        - localhost:9083
+        - --key
+        -  HIVE_METASTORE_KEYFILE
+        - --cert
+        - HIVE_METASTORE_CRTFILE
+        - --cacert
+        -  HIVE_METASTORE_CA_CRTFILE
+        - --allow-cn
+        - Any
+        image: quay.io/openshift/origin-ghostunnel
```

Comment:
A few more things:
- Can you move `image:` to be immediate after `name:` to match other containers.
- Use a variable for the image value, eg `image: "{{ .Values.presto.spec.ghostunnel.image.repository }}:{{ .Values.presto.spec.ghostunnel.image.tag }}"`.
- Also add `imagePullPolicy` to the container, also using a variable.